src/components/common/EntityInfo.vue

```diff
@@ -48,19 +41,33 @@
 
 <script lang="ts">
 // Libraries
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Mixins, Vue } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
 
 // Interfaces
 import { GetterIF } from '@/interfaces'
 
 @Component
-export default class EntityInfo extends Vue {
+export default class EntityInfo extends Mixins() {
```

Comment:
You don't need to do this if you're not importing any mixins.

Reply:
Fixed.